doc/Native APIs.md

```diff
@@ -124,6 +124,22 @@ contains these additional fields:
 
 - `active_step`: (integer) The index of the step which is currently
   active. Starts at zero.
+- `score_summary`: (object) Contains the user's score information for their completed attempts.
+  If no attempts have been completed yet, then this object will be empty (`{}`).
+  If one or more attempts have been made, then this object will contain:
+    - `correct`: (array) list of questions that were answered correctly.
+    - `correct_answers`: (integer) Number of questions answered correctly.
```

Comment:
These integer counts are a bit redundant... but that's outside of scope of this PR.